libs/wingsdk/src/target-tf-aws/counter.ts

```diff
@@ -35,7 +35,7 @@ export class Counter extends cloud.CounterBase {
     if (ops.includes(cloud.CounterInflightMethods.INC)) {
       host.addPolicyStatements({
         effect: "Allow",
-        action: ["dynamodb:UpdateItem"],
+        action: ["dynamodb:UpdateItem", "dynamodb:GetItem"],
         resource: this.table.arn,
       });
     }
```

Comment:
This condition just applies to when `inc` is called. Instead you'll want to add another if branch for the peek method:

```
    if (ops.includes(cloud.CounterInflightMethods.INC)) {
      // leave unchanged
    }
    if (ops.includes(cloud.CounterInflightMethods.PEEK)) {
      host.addPolicyStatements({
        effect: "Allow",
        action: ["dynamodb:GetItem"],
        resource: this.table.arn,
      });
    }
```

Reply:
I see, thank you @Chriscbr 

changind now